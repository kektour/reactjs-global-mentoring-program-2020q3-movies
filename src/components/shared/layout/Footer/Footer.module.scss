.root {
  height: 70px;
  background-color: #424242;
  display: flex;
}

.logoRoot {
  margin: auto;
}
