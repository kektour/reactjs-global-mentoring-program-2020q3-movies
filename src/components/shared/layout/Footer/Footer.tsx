import React from 'react';
import classnames from 'classnames';
import { Logo } from '../Logo';
import styles from './Footer.module.scss';

type Props = {
  classes?: {
    root?: string;
  };
};

export const Footer: React.FC<Props> = (props) => {
  const { classes = {} } = props;
  return (
    <div className={classnames(styles.root, classes.root)}>
      <Logo classes={{ root: styles.logoRoot }} />
    </div>
  );
};
