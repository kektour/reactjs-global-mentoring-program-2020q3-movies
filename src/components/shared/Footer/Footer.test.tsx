import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from './Footer';

describe('Footer', () => {
  it('Should render component correctly', () => {
    const component = render(<Footer />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
