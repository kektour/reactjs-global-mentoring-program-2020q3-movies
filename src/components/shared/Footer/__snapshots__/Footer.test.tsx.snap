// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render component correctly 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <p
      class="root logoRoot"
    >
      <span>
        netflix
      </span>
      roulette
    </p>
  </div>
</DocumentFragment>
`;
